lib/readline.js

```diff
@@ -1314,9 +1390,14 @@ function emitKeypressEvents(stream, iface = {}) {
 }
 
 /**
- * moves the cursor to the x and y coordinate on the given stream
+ * Moves cursor to the specified position in
+ * a given `TTY` stream.
+ * @param {Writable} stream
+ * @param {number} x
+ * @param {number} y
```

Comment:
param `y` is optional here
```suggestion
 * @param {number} [y]
```